app/Http/Requests/UserStoreRequest.php

```diff
@@ -26,4 +26,20 @@ public function rules()
             'invite_key' => ['required', 'string', 'size:4', 'exists:invite_keys,value', new InviteKeyIsAvailable()],
         ];
     }
+
+    public function messages()
+    {
+        return [
+            'username.required' => 'Vul a.u.b. een gebruikersnaam in.',
+            'username.string' => 'Vul a.u.b. een gebruikersnaam in van het type "string".',
```

Comment:
Het valt me nu pas op, maar is `van het type "string"` duidelijk voor de spelleider? Ik zou dan het woord `tekst` voorstellen. Misschien maakt `in van het type "tekst"` het weer onduidelijk 🤔 
Oke kwam ik hier nu niet met het beste voorbeeld, maar het type `string` vernoemen in een validatiefout zal onduidelijk zijn voor de spelleider.

Reply:
Op zich zou een spelleider die fout ook niet zo snel zien. Dat zou betekenen dat hij/zij actief probeert een numerieke waarde te maken (dus niet 1 invullen in een input type=text, maar van de input een type-number maken). Op dat punt lijkt me dat hij/zij weet wat ze aan het doen zijn, en dat dat niet mag volgens de server.